Marlin/src/module/temperature.cpp

```diff
@@ -1512,19 +1531,27 @@ void Temperature::disable_all_heaters() {
 
     next_max6675_ms = ms + MAX6675_HEAT_INTERVAL;
 
+//
+//	TODO: spiBegin, spiRec and spiInit doesn't work when soft spi is used.
+//		
+#if MB(MIGHTYBOARD_REVE)
     spiBegin();
     spiInit(MAX6675_SPEED_BITS);
+#endif
```

Comment:
Basing code on a particular board is bad. Only the `MIGHTYBOARD_REVE` should call these functions? It would be better to specify based on the type of SPI or a flag set by pins to indicate that the SPI needs to be treated "special."